### PR TITLE
fix: create installer should happen after generating CRDs

### PR DESCRIPTION
**Pull request by surajkota**
### What does this PR do / how does this improve the operators?
bug fix in generating CRD

### Does this PR require changes to documentation?
No

### Testing:
Make install and manifests work fine and crd get installed on cluster as expected.
Synced up with @RedbackThomson to make sure other make targets are not affected by this change

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -55,7 +55,7 @@ undeploy: manifests
 
 # Generate manifests e.g. CRD, RBAC etc.
 # Requires golang development setup for controller-gen.
-manifests: controller-gen create-installers
+manifests: controller-gen
```
Comment (mbaijal) [inline Makefile:58]: But this means we are not generating installers in `make` and `make install` at all ? The idea was to ensure these changes are checked in with PRs.

Comment (RedbackThomson) [reply Makefile:58]: Perhaps we could move what's currently in `manifests` into another target called `create-crds` and then have `manifests` be `manifests: create-crds create-installers`. That way when doing `make manifests` it creates CRDs and installers (as I suppose this change was originally intended to do).

Comment (mbaijal) [reply Makefile:58]: yep exactly!

Comment (surajkota) [reply Makefile:62]: didnt want to change the default kubebuilder make targets, created a new target generate_and_install which does as the name says

**Pull request merged**